Sources/Soto/Extensions/DynamoDB/DynamoDB+Codable.swift

```diff
@@ -371,10 +377,20 @@ extension DynamoDB {
             let expressionAttributeNames: [String: String]
             if let names = self.expressionAttributeNames, self.updateExpression != nil {
                 expressionAttributeNames = names
```

Comment:
Given you can already provide the `expressionAttributeNames` array is there a need for the `additionalAttributeNames` parameter.

Reply:
https://github.com/soto-project/soto/pull/671/commits/a1fdd48e476d4cc4fce14edf23ea0d86cbf46aa3